docs/source/tutorial_notebooks/getting_started_1_register_datasets.ipynb

```diff
@@ -270,6 +272,42 @@
     "# This is the id of the execution the updated dataset belongs to (see next tutorial)\n",
     "print(f\"Dataset assigned to execution {updated_execution_id}\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f6909cf-9826-4533-8b91-bb7a63079b37",
+   "metadata": {},
+   "source": [
+    "## 4) Special cases\n",
+    "\n",
+    "### Registering external datasets\n",
+    "\n",
+    "Typically when we register datasets we are asking the `dataregistry` to collate provenance data for the dataset and to physically manage the data (i.e., to copy the data to the central `root_dir`).\n",
+    "\n",
+    "However the `dataregistry` can also accept entries for \"external\" datasets, i.e., those located off site and not controlled by DESC. These will only be entries in the database, for querying purposes, and no physical data (managed by the `dataregistry`) will be associated with those entries.\n",
```

Comment:
I would change i.e. to e.g..
It's not a requirement that external datasets be off site, though it's a likely use case. "external" should be interpreted more or less as "external to dataregistry root_dir"